examples/sandbox/src/pages/app-search/index.js

```diff
@@ -33,12 +38,17 @@ const connector = new AppSearchAPIConnector({
     process.env.REACT_APP_SEARCH_ENDPOINT_BASE ||
     "https://search-ui-sandbox.ent.us-central1.gcp.cloud.es.io"
 });
+createTracker({
+  collectionName: "search-ui",
+  endpoint: "http://localhost:9200"
+});
 
 const config = {
   debug: true,
   alwaysSearchOnInitialLoad: true,
   apiConnector: connector,
   hasA11yNotifications: true,
+  plugins: [AnalyticsPlugin({ client: getTracker() })],
```

Comment:
could you leave app-search untouched and create a new example in`pages/elasticsearch-with-analytics` folder?